### DISCUSSION TITLE
RadioButton Styling Not Applied Due to Name Changes

### DISCUSSION THREAD

**Discussion by ManoLuuL** [primefaces/community#2378] [PrimeReact] [answered]
I'm using the RadioButton component, and when applying styling with pt using unstyled, the input part is not being applied, nor is the icon. By checking the files on GitHub, I found a change that altered the names used in the component. Therefore, it's necessary to update the names in RadioButtonPassThroughOptions as well, which is outdated and causing the passed styling to never be applied.

Link to the change: [GitHub Commit](https://github.com/primefaces/primereact/commit/1c60228ca9e2ec4e9fbc81e063c419c65c9795e2)

![image](https://github.com/user-attachments/assets/bf2ca0e7-ab27-4b70-ad96-dc1ee26abbcc)

![image](https://github.com/user-attachments/assets/3ec90796-f3f2-43eb-988c-e7b836873453)

My styling:
![image](https://github.com/user-attachments/assets/1829d1eb-c8fb-434e-8693-58ae0f459a10)

Not application:
![image](https://github.com/user-attachments/assets/59701d6a-a7b9-448a-b5fa-889a607a29ab)

**Comment by melloware** [comment]
Why not submit a PR fixing the issue if you have diagnosed it?

**Reply by ManoLuuL** [reply]
I did not take any action due to the company's policies where I work. Performing such an action would involve a bureaucratic process that would take a long time. Therefore, I decided to report the error here.

**Reply by melloware** [reply, marked as answer]
@ManoLuuL its fixed in 10.8.0 with this ticket: https://github.com/primefaces/primereact/pull/6941

IT was wrong the PT option is `rowRadioButton` and `rowCheckBox` now and they have the full Checkbox implementation
